hetmech/degree_group.py

```diff
@@ -72,13 +70,11 @@ def compute_summary_metrics(df):
 
 def dwpc_to_degrees(graph, metapath, damping=0.5):
```

Comment:
I like the new more efficient implementation (avoiding dense matrices which could cause memory problems). However, I think we want an option to yield zero-DWPC rows.